src/routes/ErrorPage.js

```diff
@@ -0,0 +1,8 @@
+export default function ErrorPage() {
```

Comment:
- [ ] Kindly, refactor this functional component to ES6 arrow function, so your code meets the latest standards. 👍